### ISSUE TITLE
[DOCS] Quickstart link accessibility

### ISSUE THREAD

**Issue by amandamartin-dev** [EddieHubCommunity/LinkFree#2523]
### Description

In the first 2 paragraphs of the linkfree quick start there are 2 links (SS below) that do not follow the same style and also are missing some accessibility features.

This issue should address the following:
Standardize link style 
Persistent underline on link (links should have a non-color indicator for accessibility)
Standard color/hover color for links (color indicator)
Standard focus style for links (keyboard only user)

This issue should also adjust any links in quickstart to open in a new tab so the user does not lose their place in the directions.  To make this accessible, warn the user in text that the link will open in a new tab.  

### Screenshots

![screenshot of linkfree quickstart with links in red squares](https://user-images.githubusercontent.com/97615019/209746783-b8918b26-52c5-4a4f-8758-c770050b165b.jpg)

**Comment by neeraj2713** [comment]
I would like to work on this issue

**Comment by neeraj2713** [comment]
Hey @amandamartin-dev , I would like to work on this issue :)

**Comment by amandamartin-dev** [comment]
Hi @neeraj2713 I do not have the ability to assign issues but you are welcome to work on this. Thank you!

**Comment by neeraj2713** [comment]
Alright!
